packages/webdriverio/src/commands/element/getValue.js

```diff
@@ -0,0 +1,35 @@
+/**
+ *
+ * Get the value of a `<textarea>`, `<select>` or text `<input>` found by given selector.
+ * If multiple elements are found via the given selector, an array of values is returned instead.
+ *
+ * <example>
+    :index.html
+    <input type="text" value="John Doe" id="username">
+    :getValue.js
+    it('should demonstrate the getValue command', () {
+        const inputUser = $('#username');
+        const value = inputUser.getValue();
+        console.log(value); // outputs: "John Doe"
+    });
+ * </example>
+ *
+ * @alias browser.getValue
+ * @param  {String} selector  input, textarea, or select element
+ * @return {String|String[]}  requested element(s) value
+ * @uses protocol/elements, protocol/elementIdProperty
+ * @type property
+ *
+ */
+
+export default function getValue () {
+    try {
```

Comment:
Let's not do a try catch here and rather use the internal flag `this.isW3C` to know which function to use.